package.json

```diff
@@ -17,7 +17,8 @@
   ],
   "author": "Grégoire Van der Auwermeulen <gregoirevandera@gmail.com>",
   "devDependencies": {
-    "bs-platform": "^7.0.1",
+    "@baransu/graphql_ppx_re": "^0.7.1",
+    "bs-platform": "^7.2.2",
     "graphql_ppx": "^0.2.8",
```

Comment:
Is there a dependency on graphql_ppx?

Reply:
Ou, it should be removed as it's replaced by graphql_ppx_re

Reply:
Can we delete?

Reply:
Pushed fix ;)